### PR TITLE
Vet Clinic Database- Query and Update Table

### PR DESCRIPTION
**Pull request by Munsa1**
# Query and Update Table

### In the following milestone, I have:

- Added a column species of type string to the animal's table.
- Inserted the required data into the database.
- Ran necessary transactions against the database
- Created the required queries against the database

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -15,3 +15,30 @@ SELECT * FROM animals WHERE neutered = 'true';
 SELECT * FROM animals WHERE animal_name != 'Gabumon';
 
 SELECT * FROM animals WHERE weight_per_kg >= '10.4' AND weight_per_kg <= '17.3';
+
+
+/* Transaction Update 1 */
+UPDATE animals SET species = 'Unspecified'
+ROLLBACK;
```
Comment (SajjadAhmad14) [inline queries.sql:22]: Please add queries before and after rollback to verify that the change was made.

File: queries.sql [hunk 2]
```diff
@@ -15,3 +15,30 @@ SELECT * FROM animals WHERE neutered = 'true';
...
+
+/* Transaction Update 1 */
+UPDATE animals SET species = 'Unspecified'
+ROLLBACK;
+
+
+/* Transaction Update 2 set species name to 'digimon' */
+UPDATE animals SET species = 'pokemon' WHERE species != 'digimon';
+
+UPDATE animals SET species = 'digimon' WHERE species = NULL;
```
Comment (SajjadAhmad14) [inline queries.sql]: Please UPDATE the animals table by setting the species column to digimon for all animals that have a name ending in mon.

File: queries.sql [hunk 3]
```diff
@@ -15,3 +15,30 @@ SELECT * FROM animals WHERE neutered = 'true';
...
 SELECT * FROM animals WHERE weight_per_kg >= '10.4' AND weight_per_kg <= '17.3';
+
+
+/* Transaction Update 1 */
+UPDATE animals SET species = 'Unspecified'
+ROLLBACK;
+
+
+/* Transaction Update 2 set species name to 'digimon' */
+UPDATE animals SET species = 'pokemon' WHERE species != 'digimon';
```
Comment (SajjadAhmad14) [inline queries.sql]: Please UPDATE the animals table by setting the species column to pokemon for all animals that have species set to NULL.

File: queries.sql [hunk 4]
```diff
@@ -15,3 +15,30 @@ SELECT * FROM animals WHERE neutered = 'true';
...
+
+UPDATE animals SET species = 'digimon' WHERE species = NULL;
+COMMIT;
```
Comment (SajjadAhmad14) [inline queries.sql:29]: Please add queries before and after commit to verify that change was made.

File: queries.sql [hunk 5]
```diff
@@ -15,3 +15,30 @@ SELECT * FROM animals WHERE neutered = 'true';
 SELECT * FROM animals WHERE animal_name != 'Gabumon';
```
Comment (SajjadAhmad14) [inline queries.sql:16]: Please add queries to find answers to the following questions:
- How many animals are there?
- How many animals have never tried to escape?
- What is the average weight of animals?
- Who escapes the most, neutered or not neutered animals?
- What is the minimum and maximum weight of each type of animal?
- What is the average number of escape attempts per animal type of those born between 1990 and 2000?

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Munsa, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: queries.sql (hunk 5 above)
Comment (Munsa1) [reply queries.sql:16]: Thank you issues have been resolved

File: queries.sql (hunk 4 above)
Comment (Munsa1) [reply queries.sql:29]: Thanks, changes implemented

File: queries.sql (hunk 3 above)
Comment (Munsa1) [reply queries.sql]: Thanks, changes implemented

File: queries.sql (hunk 2 above)
Comment (Munsa1) [reply queries.sql]: Thanks, changes implemented

File: queries.sql (hunk 1 above)
Comment (Munsa1) [reply queries.sql:22]: Thanks, changes implemented

**Review by AshakaE — APPROVED** [review]
### Hi @Munsa1   🙋‍♀️, 

# STATUS APPROVED ✔
   
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**